netci.groovy

```diff
@@ -34,6 +34,10 @@ platformList.each { platform ->
     else if (os == 'Ubuntu') {
         buildCommand = "./build.sh --skip-prereqs --configuration ${configuration} --docker ubuntu.14.04 --targets Default"
     }
+	else if (os == 'Linux') {
+		os = 'Ubuntu16.04';
```

Comment:
But I thought the whole point of this work is that it should run on any distro. So as a first test we were going to try it on Ubuntu 16.04